src/components/ModalContent/__stories__/ModalContent.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import ModalContent from "../ModalContent";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate, createStoryMetaSettings } from "../../../storybook/functions/create-component-story";
+
+export const metaSettings = createStoryMetaSettings({
+  component: ModalContent,
+  enumPropNamesArray: [], // List enum props here
+  iconPropNamesArray: [], // List props that are typed as icons here
+  actionPropsArray: [], // List the component's actions here
+});
+
+<Meta
```

Comment:
I'm not sure we need separate Stories for each SubConponent.
You can check [story of Accordion](https://62037c72e0e4d6004a9a450f-iobjldxeex.chromatic.com/?path=/docs/data-display-accordion--overview) - where you have example of Tabs for sub-item props in single story:

![Screen Shot 2022-06-26 at 12 34 22](https://user-images.githubusercontent.com/96241324/175808212-3e4bf3d4-4090-4d68-a995-56a7350aaf35.png)